stable/grafana/templates/NOTES.txt

```diff
@@ -0,0 +1,39 @@
+1. Get your '{{ .Values.server.adminUser }}' user password by running:
+
+   printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "server.fullname" . }} -o jsonpath="{.data.grafana-admin-password[*]}"`);echo
```

Comment:
Just a note that this doesnt appear to work with kubectl 1.5. Lets not change anything quite yet until we figure out a good solution that we can apply to the other charts.